### ISSUE TITLE
Independent Publisher 2: Submenu items cut off at right margin

### ISSUE THREAD

**Issue by thesacredpath** [Automattic/themes#272]
Not sure if you will choose to address this issue or not after a Slack chat with @crunnells in Themes channel, but thought I would report it just in case. @crunnells mentioned this would require JS to fix.

## Steps to reproduce

1. Set up a submenu item in IP2 and view site
2. Adjust browser window so that menu item is last item on right
3. When you hover the top level menu item so the submenu appears, it is partially cut off at edge of container

If you have more than two levels, the additional levels are not accessible unless you scroll to the right.

Screenshot:
https://cldup.com/axSz5LHZJk.png

* User report: https://en.forums.wordpress.com/topic/submenus-5/#post-3187231
* User's site: https://thebeadenmillerfamilyhistory.wordpress.com/

**Comment by allancole** [comment]
Yeah, this is a problem in a few themes that have a centered, full-width  menu. It would require some javascript to fix properly. I’ll see if I can sort it out.

**Comment by laurelfulford** [comment]
Hey @allancole! Are you still digging into this one?

**Comment by nielslange** [comment]
@thesacredpath & @allancole I've addressed this issue in https://github.com/Automattic/themes/pull/1796, thus, feel free to test that PR. 😉

**Comment by jordesign** [comment]
Confirmed this behavior is still in place.
